numpy/linalg/tests/test_linalg.py

```diff
@@ -201,21 +233,16 @@ def __repr__(self):
     LinalgCase("hmatrix_1x1",
                np.random.rand(1, 1),
                None),
-]
+])
 
 
 #
 # Gufunc test cases
 #
+def _make_generalized_cases():
```

Comment:
Is there really an advantage in having a function with no arguments that operates on a global and is called once? I think the original is cleaner and easier to follow.